apps/charterafrica/src/components/Postcard/PostCard.js

```diff
@@ -0,0 +1,62 @@
+import { Card, CardContent, CardMedia } from "@mui/material";
+import PropTypes from "prop-types";
+import { forwardRef } from "react";
+
+import LineClampedRichTypography from "../LineClampedRichTypography";
+
+const PostCard = forwardRef((props, ref) => {
+  const { author, date, image, title, sx } = props;
+
+  return (
+    <Card elevation={1} sx={sx} ref={ref}>
```

Comment:
Missing drop-shadow here compared to the design (See SpotlightCard for an example).